docs/advanced/general-privacy.md

```diff
@@ -0,0 +1,63 @@
+Either we use a centralized mixer or the later mentioned solutions, [amount correlation](https://en.bitcoin.it/wiki/Privacy#Amount) is a problem. If we send in 1.435 BTC to a mixer, and someone receives 1.434 BTC that's a quite strong indication that the transaction belongs to us.
+Amount correlation will be a problem even if we use a so called _black box privacy technique_, like sending in our money to an anonymous exchange, exchanging it to a privacy coin which uses [zero-knowledge proof](https://en.wikipedia.org/wiki/Zero-knowledge_proof), and then switching back the money to our original currency. Amount correlation can still lead to loss of privacy. One can address this issue by breaking up the total balance to random chunks and distributing it to newly generated addresses, over a longer period of time.
+
+## Non-custodial coin mixing
+
+It is possible to mix coins without a centralized exchange. One method is [CoinJoin](https://en.bitcoin.it/wiki/CoinJoin), which will combine multiple payments from multiple spenders into a single transaction.
+Other method is [CoinSwap](https://en.bitcoin.it/wiki/CoinSwap), which is using atomic swaps. A third method is [TumbleBit](https://en.bitcoin.it/wiki/TumbleBit), which allows parties to make payments through an untrusted Tumbler.
+[Decred's build in coin mixing](https://blog.decred.org/2019/08/28/Iterating-Privacy/) feature is based on the CoinShuffle++ protocol.
+
+## Using the Lightning Network for privacy
+
+Privacy attacks like [common-input-ownership heuristic](https://en.bitcoin.it/wiki/Privacy#Common-input-ownership_heuristic), [address reuse](https://en.bitcoin.it/wiki/Privacy#Address_reuse) and change address detection fundamentally don’t work on any off-chain transactions technology. The Lightning Network is an off-chain transaction technology. One possible downside is that the operator of the lightning node could keep logs.
+
+
+## Hiding cryptocurrency usage itself
+
+Use VPN or Tor. Don’t link your VPN or Tor usage with social networks, or with any other software or website that might identify you. Keep in mind that cookies exist. Use a browser profile that is dedicated to your crypto-related activities only. Quit all programs that you don’t need. Download some random data, so the size of your downloaded data won’t be exactly blockchain-sized. It’s a good idea to use virtual machines.
+
+## Platform-related questions
+
+Keep in mind that the platform you are using can also have vulnerabilities. A VPN does not provide any means of defense if the system itself that you are using is hacked. Try to use open-source software, and if possible, open-source hardware as well. See [General-Security](general-security) for more information.
```

Comment:
What do you mean by platform here? Operating system?

Link to `general-security.md` with markdown file extension.